react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  React is a front end javascript libary for building user interfaces based on components
+- What is a React component?
+  isolated pieces of UI that have their own functionality that can be re-used
+- How do you mount a React app (root component) to the DOM?
+  npm run dev
```

Comment:
Not quite! That's how we start our server. How are we actually connecting React to our HTML?